docs/pages/access-controls/sso/okta.mdx

```diff
@@ -1,107 +1,131 @@
+single sign on (SSO) identities to Teleport Enterprise and Teleport Enterprise
+Cloud. When used in combination with role based access control (RBAC), this allows
+Teleport administrators to define policies like:
 
-- Only members of "DBA" group can SSH into machines running PostgreSQL.
+- Only members of "DBA" group can access into PostgreSQL databases.
 - Developers must never SSH into production servers.
+- Members of the HR group can view audit logs but not production environments.
 
 ## Prerequisites
 
-- Okta account with admin access. Your account must include users and at least two groups.
-- Teleport role with access to maintaining `saml` resources.  This is available in the default `editor` role.
+- An Okta account with admin access. Your account must include users and at
+  least two groups.
 
 (!docs/pages/includes/commercial-prereqs-tabs.mdx!)
 
+- A Teleport role with access to edit and maintain `saml` resources. This is
+  available in the default `editor` role.
+
 (!docs/pages/includes/tctl.mdx!)
 
 (!docs/pages/includes/enterprise/samlauthentication.mdx!)
 
-## Configure Okta
+## Step 1/4. Create & assign groups
+
+### Create Groups
+
+Create two groups: "okta-dev" and "okta-admin". In the Okta dashboard go to the
+navigation bar and click **Directory** -> Groups, then **Add group**:
+
+![Create Group Devs](../../../img/sso/okta/okta-saml-2.1@2x.png)
+
+Repeat for the admin group:
+
+![Create Group Devs](../../../img/sso/okta/okta-saml-2.2.png)
+
+## Step 2/4. Configure Okta
 
-First, create a SAML 2.0 Web App in Okta configuration section
+First, create a SAML 2.0 Web App in Okta:
 
-### 1. Select Create New App
+From the main navigation menu, select **Applications** -> Applications, and click
+**Create App Integration**. Select SAML 2.0, then click **Next**.
 
-Select Web Platform and SAML 2.0 Sign On Method.
 ![Create APP](../../../img/sso/okta/okta-saml-1.png)
 
-## Configure the App
+On the next screen (**General Settings**), provide a name and optional logo for your new app, then
+click **Next**. This will bring you to the **Configure SAML** section.
+
+### Configure the App
 
-We are going to map the Okta groups we've created above to the SAML Attribute
-statements (special signed metadata exposed via a SAML XML response).
+Provide the following values to their respective fields:
 
-GENERAL
+#### General
 
-- Single sign on URL `https://<cluster-url>/v1/webapi/saml/acs/new_saml_connector`
-- Audience URI (SP Entity ID)`https://<cluster-url>/v1/webapi/saml/acs/new_saml_connector`
+- Single sign on URL: `https://<cluster-url>/v1/webapi/saml/acs/new_saml_connector`
+- Audience URI (SP Entity ID): <nobr>`https://<cluster-url>/v1/webapi/saml/acs/new_saml_connector`</nobr>
 - Name ID format `EmailAddress`
 - Application username `Okta username`
 
-SINGLE ATTRIBUTE STATEMENTS
+Replace `<cluster-url>` with your Teleport proxy address or cloud tenant (e.g. `mytenant.teleport.sh`).
```

Comment:
I use Okta for my home cluster's IdP without defining a port in the Okta Application setting. You can define one, but since the Teleport Proxy service can't be configured to listen on alternate ports (at least since the last time I asked), It doesn't matter if you do or don't include `:443`.